src/styleUtils.js

```diff
@@ -108,37 +106,14 @@ define(
 						});
 
 						saveButton.children().remove();
-
-						saveButton.downloadify({
-							swf : '../external/downloadify/downloadify.swf',
-							downloadImage : '../external/downloadify/download.png',
-							width : 100,
-							height : 30,
-							filename : filename,
-							data : cslCode,
-							transparent : true,
-							onComplete: function () {
-								alert('Your CSL Style Has Been Saved');
-								dialog.dialog('destroy');
-							},
-							onCancel: function () { },
-							onError: function () { alert('Error saving file.'); }
+						
+						dialog.find('#download').html('<button id="download-button">Save To Disk</button>');
```

Comment:
(BTW is "Save to Disk" nowadays still the best choice for the label here? My association would be a floppy disk...)